Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/EntityPropertyEditor.cpp

```diff
@@ -3942,18 +4005,16 @@ namespace AzToolsFramework
         {
             m_gui->m_componentList->ensureWidgetVisible(componentEditor);
         }
-        m_shouldScrollToNewComponents = false;
-        m_shouldScrollToNewComponentsQueued = false;
         m_newComponentId.reset();
 
         HighlightMovedRowWidget();
     }
 
     void EntityPropertyEditor::QueueScrollToNewComponent()
     {
-        if (m_shouldScrollToNewComponents && !m_shouldScrollToNewComponentsQueued)
+        if (!m_shouldScrollToNewComponentsQueued)
         {
-            QTimer::singleShot(100, this, &EntityPropertyEditor::ScrollToNewComponent);
+            QTimer::singleShot(30, this, &EntityPropertyEditor::ScrollToNewComponent);
```

Comment:
Should comment on any scenario where this number (30) should be adjusted in the future?

Reply:
Yeah, I don't like that this number exists, a value of 1 might actually work here.  It just has to be "when things settle" and technically, the only wrong value is 0.
But I'll add a comment.